framework/source/class/qx/ui/form/core/VirtualDropDownList.js

```diff
@@ -469,6 +469,10 @@ qx.Class.define("qx.ui.form.core.VirtualDropDownList",
     __getAvailableHeight : function()
     {
       var distance = this.getLayoutLocation(this._target);
+      if (!distance) {
```

Comment:
Returning zero from here means that `__adjustHeight` ([see here](https://github.com/milandamen/qooxdoo/blob/d0f02bdb691e0d622f1ba8a5303ce8da8e43321b/framework/source/class/qx/ui/form/core/VirtualDropDownList.js#L451-L453)) will change the value of the `maxHeight` property.  The recent PR https://github.com/qooxdoo/qooxdoo/pull/9377 has demonstrated that the side effects of changing properties (specifically minHeight/maxHeight) can cause problems.

A better response would be to return a non-numeric value from `__getAvailableHeight` (eg `null` or `undefined`) and make `__adjustHeight` immediately return if that value is detected.

Reply:
Thank you for the suggestion, I have just implemented that.